src/Options.js

```diff
@@ -181,6 +185,16 @@ Options.prototype.baseSizingClasses = function() {
 };
 
 /**
+ * Return the font metrics for this size.
+ */
+Options.prototype.fontMetrics = function() {
+    if (!this._fontMetrics) {
+        this._fontMetrics = fontMetrics.getFontMetrics(this.size);
```

Comment:
When I first looked at this I thought this might create a lot of objects but then I remembered that `getFontMetrics` also stores a copy of each unique object it returns which means there's a small number of objects being created.